articles/architecture-scenarios/b2c/tenant-architecture.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Tenant Architecture
+description: Learn how to create Auth0 tenants for one or more environments
+toc: true
+topics:
+    - b2c
+    - ciam
+    - tenants
+contentType: concept
+useCase:
+  - tenant-architecture
+---
+# Tenant Architecture
+
+Everything at Auth0 begins with the creation of a tenant. The tenant is where you will configure everything related to your use of Auth0.
+
+You can (and are encouraged to) create more than one tenant. In addition to using different tenants for different environments (for example, you can have a tenant for development and a tenant for production), you can use different tenants to isolate your user domains (determining which user domains you'll need is an important step of the tenant architecture process).
+
+::: warning
+Tenant names cannot be changed or reused (even if the original tenant has been deleted). Make sure that you are happy with your names before you set up the tenants.
+:::
+
+## Design considerations
```

Comment:
Is this documentation constructed from the latest B2C Welcome provided? I notice the section is using numbered bullet points; the original B2C Welcome section ( https://docs.google.com/document/d/1wHEmRsPVciJJVuyssBH3RyO1mLYIGqLj34-vhVFEODU/edit#bookmark=id.xmqjgu35ensx) was changed to favour non-numbered bullet points.